vector/src/main/java/im/vector/riotx/features/home/room/list/RoomListViewModel.kt

```diff
@@ -101,7 +101,18 @@ class RoomListViewModel @Inject constructor(initialState: RoomListViewState,
                 .observeOn(Schedulers.computation())
                 .map { buildRoomSummaries(it) }
                 .execute { async ->
-                    copy(asyncFilteredRooms = async)
+                    val invitedRooms = async()?.get(RoomCategory.INVITE)?.map { it.roomId } ?: emptyList()
+                    val remainingJoining = joiningRoomsIds.intersect(invitedRooms)
+                    val remainingRejecting = rejectingRoomsIds.intersect(invitedRooms)
+                    val remainingRejectErrors = rejectingErrorRoomsIds.intersect(invitedRooms)
+                    val remainingJoinErrors = joiningErrorRoomsIds.intersect(invitedRooms)
+                    copy(
+                            asyncFilteredRooms = async,
+                            joiningRoomsIds = remainingJoining,
+                            rejectingRoomsIds = remainingRejecting,
+                            rejectingErrorRoomsIds = remainingRejectErrors,
+                            joiningErrorRoomsIds = remainingJoinErrors
```

Comment:
About the lisibility of code, the lines could be reordered, when declaring the 4 `val` and here

Reply:
done